infrastructure/tasks/00-cosmosdb.ts

```diff
@@ -101,17 +101,17 @@ export const run = async () => {
   );
 
   const databaseAccount = await client.databaseAccounts.get(
-    (config as any).azurerm_resource_group_00,
-    (config as any).azurerm_cosmosdb_00
+    (config as any).azurerm_resource_group,
```

Comment:
sure about this? keep in mind you're handling production infrastructure here, you want to be super super super super sure `config` is valid and `azurerm_resource_group` is valid too... :)

Reply:
what do you mean exactly with "valid" ? =)

Reply:
I mean, I guess you expect `azurerm_resource_group` is a non-empty string right? There's no guarantee here that that's what you'll get (this is true for all references to `config` in this file).

With Terraform I can run the `plan` command and have TF verify that everything is in place for and good. I can confidently run `apply` knowing that there will not be problems (at least not problems I can control).